docs/maui/behaviors/index.md

```diff
@@ -19,5 +19,6 @@ The .NET MAUI Community Toolkit provides a collection of pre-built, reusable beh
 
 | Converter | Description |
 | --------- | ----------- |
+| [`MaskedBehavior`](masked-behavior.md) | The `MaskedBehavior` is a `Behavior` that allows the user to define an input mask for data entry. |
```

Comment:
We don't `Behavior` the rest in this table as well. I'm fine with either, but I think we should make it consistent at least :)